src/HonzaBotner.Discord.Services/Commands/VoiceCommands.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Threading.Tasks;
+using DSharpPlus;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using DSharpPlus.Entities;
+using HonzaBotner.Discord.Services.Attributes;
+
+namespace HonzaBotner.Discord.Services.Commands
+{
+    [Group("voice")]
+    [Description("Commands to control custom voice channels.")]
+    [RequireChannel(750108543125946448)]
```

Comment:
Maybe extract this value into constant
```suggestion
    [RequireChannel(VoiceCommands.RequiredChannel)]
```
where you can comment what does this Value mean.

However, I dont really like this hardcoded version, but I dont know better way right now.